contracts/StakeManager.sol

```diff
@@ -54,6 +54,8 @@ contract StakeManager is
     uint256 public feeBps; // range {0-10_000}
     mapping(uint256 => bool) public rewardsIdUsed;
 
+    address public whitelistedAccount;
```

Comment:
Maybe `whitelistedAddress` or `redirectAddress` or something that explains the logic might be better.

Reply:
redirectAddress is better

Reply:
okay sure, will update